autoscaling.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_launch_configuration" "asg_launch" {
+  name_prefix     = "asg-launch"
+  image_id        = "ami-068257025f72f470d"
+  instance_type   = "t3.small"
+  # user_data       = file("user_data.sh")
+  security_groups = [aws_security_group.lb_sg.id, aws_security_group.bastion_host_server_sg.id]
```

Comment:
Why is the launch configuration of the ASG using the SG of the loadbalancer and the bastion ? What is the reason behind this ?